python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/dbt_packages/test_columns_metadata.py

```diff
@@ -177,10 +177,15 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
         "trino",
     ],
 )
+@pytest.mark.parametrize(
+    "defer_metadata_collection_and_lineage_processing",
+    [True, False],
+)
 def test_column_lineage(
     test_metadata_manifest: Dict[str, Any],
```

Comment:
I think we should test the experimental `fetch_column_schema` functionality on the vanilla jaffle shop project that doesn't have the `dagster` dbt package enabled on it at all. 

So we should add `test_jaffle_shop_manifest` as a fixture on this function.

Reply:
It looks like the jaffle shop project has a few subtle structural differences so that the assertions would need to be adjusted. For now, I added an env var to the metadata project that disables the macro if set, I think that ends up 
 with cleaner tests at the expense of making the project.yaml a bit thornier.